test/dates/adjusters.jl

```diff
@@ -183,6 +183,17 @@ end
 @test Dates.lastdayofquarter(Dates.DateTime(2014,8,2)) == Dates.DateTime(2014,9,30)
 @test Dates.lastdayofquarter(Dates.DateTime(2014,12,2)) == Dates.DateTime(2014,12,31)
 
+first = Dates.Date(2014,1,1)
```

Comment:
I'd just rename the variables from `first` and `last` to `first_dt` and `last_dt`.